client/my-sites/promote-post-i2/components/post-item/index.tsx

```diff
@@ -126,10 +137,22 @@ export default function PostItem( { post }: { post: BlazablePost } ) {
 			</td>
 
 			<td className="post-item__post-type">{ getPostType( post.type ) }</td>
-			<td className="post-item__post-publish-date">{ postDate }</td>
-			<td className="post-item__post-views">{ formatNumber( viewCount, true ) }</td>
-			<td className="post-item__post-likes">{ formatNumber( likeCount, true ) }</td>
-			<td className="post-item__post-comments">{ formatNumber( commentCount, true ) }</td>
+			{ post.type !== 'product' || ! isRunningInWooStore ? (
```

Comment:
As I mentioned before, I think it Will be easy if we can group the changes without using else. It will make us not having duplicate code for the columns. 

It is not wrong the way it is implemented, but I think it will be better only have a single instance of `<td className="post-item__post-views">{ formatNumber( viewCount, true ) }</td>`. It will help us in the future if we need to change that column. It will be in a single place.

Reply:
got it :) will change it

Reply:
Fixed